docs/pages/repo/docs/reference/system-variables.mdx

```diff
@@ -13,6 +13,7 @@ By setting certain environment variables, you can change Turborepo's behavior. T
 | `TURBO_BINARY_PATH`                | Manually set the path to the `turbo` binary. By default, `turbo` will automatically discover the binary so you should only use this in extremely rare circumstances.                                                                          |
 | `TURBO_CI_VENDOR_ENV_KEY`          | Set a prefix for environment variables that you want **excluded** from [Framework Inference](/repo/docs/core-concepts/caching/environment-variable-inputs#framework-inference).                                                               |
 | `TURBO_FORCE`                      | Always force all tasks in your pipelines to run in full, opting out of all caching.                                                                                                                                                           |
+| `TURBO_CACHE_DIR`                  | Sets the cache directory, similarly to calling `--cache-dir`-argument                                                                                                                                                                         |
```

Comment:
Not sure how you want to sort/order this table

Reply:
Alphabetical would be great! So up two. 👍